server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueueTest.java

```diff
@@ -153,6 +155,25 @@ public ManageableMailQueue getManageableMailQueue() {
         return mailQueueFactory.createQueue(SPOOL);
     }
 
+    @Test
+    void browseShouldReturnMailsWithMimeMessage() throws Exception {
```

Comment:
Is this specific to RabbitMQ?

I mean, this was missed as part of the original implementation because `ManageableMQContract` never checked that.

Could wemove this in the contract? This seems important...